src/client/testing/page-objects/create-availability/summary-page.ts

```diff
@@ -0,0 +1,21 @@
+import { type Locator, type Page } from '@playwright/test';
+import RootPage from '../root';
+import { DatabaseAccount } from '@azure/cosmos';
+
+export default class SummaryPage extends RootPage {
+  readonly summaryPageTitle: Locator;
+  readonly saveSessionButton: Locator;
+
+  constructor(page: Page) {
+    super(page);
+    this.summaryPageTitle = page.getByText('Check single date session');
+
+    this.saveSessionButton = page.getByText('Save session');
+  }
+
+  async changeFunctionalityLink(val: string) {
+    const row = this.page.locator('.nhsuk-summary-list__row', { hasText: val });
+    const changeLink = row.locator('.nhsuk-summary-list__actions a');
+    await changeLink.click();
+  }
```

Comment:
We should avoid using classname selectors as this isn't how screen readers interact with the DOM. If we can't find an element using roles / labels / visible text, then there's a good chance screen readers can't either. 

The summary list uses `<dl>` elements which, [according to the docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl#technical_summary), can be given the role `list`. I think therefore we should make the following changes to `summary-list.tsx`: 
<img width="371" alt="image" src="https://github.com/user-attachments/assets/d4c5bb35-6064-40f5-af31-5ac7ff0efc98" />

and then rewrite this method to be something along the lines of:

<img width="390" alt="image" src="https://github.com/user-attachments/assets/adf3ea94-ce45-448d-b651-ff772180f84c" />
(that may not work exactly, some tweaking may be required)